src/archive/2022-2023/demos/3/README.md

```diff
@@ -0,0 +1,61 @@
+# Demo #3, April 6th
+
+## Spirit
+
+1. Do a final showcase simulation.
+1. Show the world what you've built.
+1. See what other teams have built.
+1. No matter what, enjoy the process.
+
+   The amount of hard work, dedication,
+   and teamwork required to bring an idea to life is immense,
+   and you should all feel incredibly proud
+   of what you have accomplished in the last two seasons.
+   You have overcome countless obstacles and challenges,
+   honed your skills, and become a better version of yourself.
+   Now, as you stand at the finish line
+   and look back on what you have achieved,
+   take a moment to appreciate how far you have come
+   and how much you have grown.
+   Your project is a testament to your passion, creativity, and perseverance,
+   and it serves as a reminder that with the right mindset and determination,
+   anything is possible.
+
+## Agenda
+
+The content is up to you!
+The way you deliver your presentation matters,
+so feel free to drop here your signature style.
+
+Just remember that in the final showcase
+(and therefore in this demo),
+you'll have:
+
+- Maximum 15 minutes for your presentation.
+- Maximum 3 minutes for questions and answers.
+- Most of the audience will be completely new to your project.
+- Teams will present in the following order:
+
+  1. Bandist.

```

Comment:
Update list of teams

Reply:
These are in the archive, so I kept them the same as last year 👁️